Sources/WordPressData/Swift/ReaderTeamTopic.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+@objc(ReaderTeamTopic)
+open class ReaderTeamTopic: ReaderAbstractTopic {
```

Comment:
Moved from `Keystone`.

Reply:
Ouch. I was sure to have moved all the types but clearly missed this. I think the fact that it does not inherit from `NSManagedObject` tricked me.

Reply:
I greped the model and checked every `representedClassName` – we should be good now.